electron/electron.ffmpeg.js

```diff
@@ -0,0 +1,83 @@
+/*
+    FFMpeg wrapper
+*/
+
+import log from 'electron-log';
+import { spawn } from 'child_process';
+import { main } from '../config/config';
+
+const { rtmpPort, frameConfig } = main;
+
+/* track if user stopped the process */
+let userStopFFmpeg = false;
+
+const startFFMpeg = (sender, rtmpStrmID, configIdx = 0) => {
+    log.info(`Launching FFmpeg with config: ${configIdx}`)
+    userStopFFmpeg = false  /* reset !!*/
+
+    const framerate = frameConfig[configIdx].framerate
+    const keyint = frameConfig[configIdx].keyint
+    const FFMPeArgs = [
+        '-f', 'avfoundation',
+        '-framerate', framerate,
+        '-pixel_format', 'uyvy422',
+        '-i', '0:0',
+        '-vcodec', 'libx264',
+        '-tune', 'zerolatency',
+        '-b', '900k',
+        '-x264-params', `keyint=${keyint}:min-keyint=${keyint}`,
+        '-acodec', 'aac',
+        '-ac', '1',
+        '-b:a', '96k',
+        '-f', 'flv',
+        `rtmp://localhost:${rtmpPort}/stream/${rtmpStrmID}`];
+
```

Comment:
Still don't know what happened, so I tried with the new release (https://github.com/livepeer/go-livepeer/releases) and it worked with the command line, so I tried to change a few stuff to match the new api and it seem to work, I guess when the new binary will be uploaded + the endpoints fixed this error will be fixed.